module/accelerometer/include/accelerometer.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023 Robarm
+// All rights reserved
+#ifndef MODULE_ACCELEROMETER_INCLUDE_ACCELEROMETER_HPP_
+#define MODULE_ACCELEROMETER_INCLUDE_ACCELEROMETER_HPP_
+
+#include "hal/i2c/include/i2c_component.hpp"
+
+struct AccelerationRAW_t {
+    int16_t x;
+    int16_t y;
+    int16_t z;
+};
+
+struct Acceleration_t {
+    float x;
+    float y;
+    float z;
+};
+
+namespace robarm {
+namespace module {
+namespace accelerometer {
+class Accelerometer {
+private:
+    robarm::hal::i2c::I2C_Component i2c_;
+    AccelerationRAW_t accelRaw_;
+    Acceleration_t accel_;
+
+public:
+    Accelerometer();
+    Accelerometer(robarm::hal::i2c::I2C_Bus bus);
```

Comment:
One parameter constructor can be made explicit. Also, since this class is already inside a robarm namespace you can ommit it.
```suggestion
    explicit Accelerometer(hal::i2c::I2C_Bus bus);
```